### PR TITLE
Synchronize blockgroup and production state from Blocks

### PR DESCRIPTION
**Pull request by zachthompson**
This updates the blockgroup and production_state in the metadata to match Blocks.  Needs to be coordinated with internal PR.

cc @MariagraziaAlastra

### CODE REVIEW

File: sql/1.153/sync-blockgroups.sql [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+		('DDG::Goodie::Uppercase', 'goodie'),
+		('DDG::Goodie::Uptime', 'goodie'),
+		('DDG::Goodie::URLEncode', 'goodie'),
+		('DDG::Goodie::ValarMorghulis', 'goodie'),
+		('DDG::Goodie::Wavelength', 'goodie'),
+		('DDG::Goodie::Week', 'goodie'),
+		('DDG::Goodie::WeekdaysBetween', 'goodie'),
+		('DDG::Goodie::Weight', 'goodie'),
+		('DDG::Goodie::WhereAmI', 'goodie'),
+		('DDG::Goodie::WorkdaysBetween', 'goodie'),
```
Comment (moollaza) [inline sql/1.153/sync-blockgroups.sql]: Look like a duplicate!

Comment (moollaza) [inline sql/1.153/sync-blockgroups.sql]: Whoops, nevermind. `Work` vs `Week`

**Comment by zachthompson** [comment]
We good @moollaza @MariagraziaAlastra?  Looks like the last Blocks change was 7 days ago and is in here.  We're reviewing the internal PR this morning.

**Comment by MariagraziaAlastra** [comment]
Yeah, just tested, LGTM :+1: 
Feel free to merge when the internal PR is ready :smile:

**Comment by zachthompson** [comment]
Thanks.  As long as @moollaza isn't planning any Blocks changes, we can merge at any time.  The updates won't affect anything.

**Pull request merged**